Tools/video_matching_eval.cpp

```diff
@@ -79,6 +92,9 @@ string getDateTime();
  * - Waits for minimum amount of time and displays the frame
```

Comment:
Sorry I think I left this in when I used the video profiler comments, do you mind changing this to Detection Evaluation or something similar?